File: postgre/scala/delete/postgre_delete.scala
// -----------------------------------------------------------------
//	delete/postgre_delete.scala
//
//						Jul/31/2011
// -----------------------------------------------------------------
import scala.io.Source
import java.sql._

// -----------------------------------------------------------------
object postgre_delete
{
// -----------------------------------------------------------------
def main (args: scala.Array[String])
{
	println	("*** 開始 ***")

	val id_in = args(0)

	println (id_in)

	postgre_delete_proc (id_in)

	println	("*** 終了 ***")
}

// -----------------------------------------------------------------
def postgre_delete_proc (id_in:String)
{
	val str_connect= "jdbc:postgresql://localhost/city"

	val conn = DriverManager.getConnection (str_connect,"scott","tiger")

	sql_manipulate.delete_proc (conn,id_in)
	sql_manipulate.display_proc (conn)

	conn.close
}

// -----------------------------------------------------------------
}
// -----------------------------------------------------------------

File: postgre/java/delete/postgre_delete.java
// ----------------------------------------------------------------------
/*
	java/delete/postgre_delete.java

				May/26/2011


*/
// ----------------------------------------------------------------------
import java.sql.Connection;
import java.sql.DriverManager;

// ----------------------------------------------------------------------
public class postgre_delete
{

// ----------------------------------------------------------------------
public static void main (String[] args) throws Exception
{
	System.out.println ("*** 開始 ***");

	String	id = args[0];
	System.out.println ("\tid = " + id);

	Connection conn =
		DriverManager.getConnection
			("jdbc:postgresql://localhost/city","scott","tiger");

	rdb_common.delete_proc	(conn,id);

	rdb_common.display_proc	(conn);

	conn.close ();

	System.out.println ("*** 終了 ***");
}

// ----------------------------------------------------------------------
}
// ----------------------------------------------------------------------
